torch/csrc/api/src/nn/options/linear.cpp

```diff
@@ -9,5 +9,11 @@ LinearOptions::LinearOptions(int64_t in_features, int64_t out_features)
 BilinearOptions::BilinearOptions(int64_t in1_features, int64_t in2_features, int64_t out_features)
   : in1_features_(in1_features), in2_features_(in2_features), out_features_(out_features) {}
 
+UnflattenOptions::UnflattenOptions(int64_t dim, std::vector<int64_t> sizes)
+  : dim_(dim), sizes_(std::move(sizes)) {}
+
+UnflattenOptions::UnflattenOptions(std::string dimname, namedshape_t namedshape)
```

Comment:
dim_ is primitive type, need a initial value, either initialize here or inside class doing int64_t dim_ = <init value, eg. 0> directly